packages/salesforcedx-vscode-apex/src/apexLspStatusBarItem.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as vscode from 'vscode';
+
+// TODO Use localization?
+const APEX_LSP_LABEL = 'Apex LSP';
```

Comment:
If this ends up staying, we'll want to localize it - let's add it to the [localization file here](https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-apex/src/messages/i18n.ts)